### DISCUSSION TITLE
SLA Notification

### DISCUSSION THREAD

**Discussion by Dkryptur** [DefectDojo/django-DefectDojo#10412] [Q&A]
I have created notification, and while testing I created one Critical vulnerabilities, but I am not getting notification on it.
can anyone tell me after how much time these mails are being sent.
![image](https://github.com/DefectDojo/django-DefectDojo/assets/53573052/8840420e-ab12-4b80-959c-cd9783554c79)

**Comment by Dkryptur** [comment]
can anyone help me on this ?

**Reply by mtesauro** [reply]
I get email alerts from DefectDojo quickly (a minute or two at most) - I'm assuming you have setup email/SMTP correctly.  If not, that's a good place to start debugging this. Also, look in the logs for any errors when you try to trigger them being sent.

Also, another good place to ask is OWASP's Slack instance on #defectdojo - see https://github.com/DefectDojo/django-DefectDojo?tab=readme-ov-file#community-getting-involved-and-updates details on joining.

**Reply by Dkryptur** [reply]
Even I am getting mails for other things like product add, test add. but not for SLA breach.

**Reply by mtesauro** [reply]
If you make a finding breach an SLA on purpose, does anything show up in the logs? I'd look in the celery and uwsgi logs particularly
